### ISSUE TITLE
Full working example for SmoothCollapsingToolbarLayout

### ISSUE THREAD

**Issue by kuwapa** [henrytao-me/smooth-app-bar-layout#124]
Hello @henrytao-me , 

The library seems to be awesome. But I'm not able to make SmoothCollapsingToolbarLayout from the code on readme work for me. I couldn't find a full working example of SmoothCollapsingToolbarLayout in your sample app either. 

Would be super helpful if you could provide an example. 

Thanks

**Comment by henrytao-me** [comment]
Hi @abhiank209 

Can you tell me what AppCompat and Support Design version are you using?

**Comment by david-perez** [comment]
@henrytao-me I also find it incredibly difficult to integrate this library with the information provided in the readme.

Especially given the fact that all of the sample's examples make use of third party libraries such as custom [RecyclerView's adapters](https://github.com/henrytao-me/recyclerview-multistate-section-endless-adapter/tree/master/recyclerview/src/main/java/me/henrytao/recyclerview) and [design libraries](https://github.com/henrytao-me/android-md-core), which I do not want to include. The sample should provide a fully working example without these additional libraries, otherwise these libraries should be declared explicitly as required dependencies in the readme.

For example, using only this library and what I was able to understand/copy from the readme and the sample, I came up with a minimal app which I've put in [this](https://github.com/david-perez/SmoothAppBarLayoutProofOfConcept) repository.

The result is disappointing, [here](http://sendvid.com/xrht0a4y) is a video. I don't know why the jitters are there.
